app/src/main/java/com/daclink/drew/sp22/cst438_project01_starter/User.java

```diff
@@ -9,9 +9,9 @@
     private String firstName;
     private String lastName;
     private String location;
-    @PrimaryKey(autoGenerate = true)
+    @PrimaryKey
```

Comment:
Hey, I think this should be left as **@PrimaryKey(autoGenerate = true)** because this helps when we insert an object into the database. It autogenerates a key for that object passed in.